src/ripple/app/main/Application.cpp

```diff
@@ -1126,6 +1138,7 @@ bool ApplicationImp::setup()
             getWalletDB (), "PublisherManifests");
 
         m_networkOPs->setValidationKeys (valSecret, valPublic);
+        setMaxLedger();
```

Comment:
I'd rather keep this broken out as a separate function because the calling function is so big already.